Cargo.toml

```diff
@@ -20,7 +20,7 @@ description = "Neqo, the Mozilla implementation of QUIC in Rust."
 keywords = ["quic", "http3", "neqo", "mozilla", "ietf", "firefox"]
 categories = ["network-programming", "web-programming"]
 readme = "README.md"
-version = "0.12.1"
+version = "0.12.2"
```

Comment:
Note the version bump. This should be the only missing piece for landing Neqo v0.12 in mozilla-central (https://phabricator.services.mozilla.com/D234901).